diff --git a/node_modules/react-native-actions-sheet/dist/src/hooks/use-scroll-handlers.js b/node_modules/react-native-actions-sheet/dist/src/hooks/use-scroll-handlers.js
index xxxxxxx..xxxxxxx 100644
--- a/node_modules/react-native-actions-sheet/dist/src/hooks/use-scroll-handlers.js
+++ b/node_modules/react-native-actions-sheet/dist/src/hooks/use-scroll-handlers.js
@@ -102,6 +102,7 @@ export function useScrollHandlers(options) {
             var _a;
             var ref = resolveScrollRef(nodeRef);
             if (Platform.OS == 'web') {
+                if (!ref) return;
                 var rect = ref.getBoundingClientRect();
                 ref.style.overflow = "auto";
                 onMeasure(rect.x, rect.y, rect.width, rect.height, rect.left, rect.top);